components/server/ee/src/prebuilds/bitbucket-server-app.ts

```diff
@@ -102,6 +102,12 @@ export class BitbucketServerApp {
             const cloneUrl = context.repository.cloneUrl;
             const commit = context.revision;
             const projectAndOwner = await this.findProjectAndOwner(cloneUrl, user);
+            if (projectAndOwner.project) {
+                /* tslint:disable-next-line */
+                /** no await */ this.projectDB.updateProjectUsage(projectAndOwner.project.id, {
```

Comment:
why no await here?

Reply:
Good question. This is how every other `updateProjectUsage` call site does it -- the thought was something like: This is not important right now, so it's fine to do it async without delaying the current prebuild/workspace start.